### PR TITLE
increase value of `MAX_SLOTS_TO_REMEMBER` to 14400

### LINKED ISSUE #354: Increase time window for fetching SCP messages from overlay
To increase the time window where our vault clients can fetch SCP messages from the overlay instead of from archive nodes, we will change the [MAX_SLOTS_TO_REMEMBER](https://github.com/stellar/stellar-core/blob/ca2fb06059c15442cb4c9a8c89de1a8fc3579a39/docs/stellar-core_example.cfg#L478) to **`14400`**.
By changing it to `14400` we increase the time window to 24 hours.

We need to update [this](https://github.com/pendulum-chain/spacewalk/blob/befd6fe8de903407e212902956398df25b50e8f3/clients/vault/src/oracle/constants.rs#L19) constant to that same value once done.

### PR DESCRIPTION
**Pull request by b-yap**
closes #354

### CODE REVIEW

**Review by ebma — CHANGES_REQUESTED** [review]
We'll need to increment this value even further, just not yet sure to what exactly.

**Comment by TorstenStueber** [comment]
As a reference: https://github.com/pendulum-chain/spacewalk/issues/354

**Comment by ebma** [comment]
I updated the original ticket. We want to increase it to `14400` now. But we'll first have to check if the archive nodes break down because of that.

File: clients/vault/src/oracle/constants.rs [hunk 1]
```diff
@@ -16,7 +16,8 @@ pub const MAX_DISTANCE_FROM_CURRENT_SLOT: Slot = 3;
 pub const VALIDATOR_COUNT_TEST_NETWORK: usize = 3;
 pub const VALIDATOR_COUNT_PUBLIC_NETWORK: usize = 23;
 
-pub const MAX_SLOT_TO_REMEMBER: Slot = 12;
+/// increase the time window to 24 hours.
+pub const MAX_SLOTS_TO_REMEMBER: Slot = 14400;
```
Comment (ebma) [inline clients/vault/src/oracle/constants.rs:20]: Nice. Let's maybe rewrite the comment though to give a bit more context.
```suggestion
/// Set the _expected_ `MAX_SLOTS_TO_REMEMBER` parameter to 14400 Slots on Stellar, 
/// ie. 24 hours. This parameter is only correct when connecting to SatoshiPay Stellar validators
/// as the configuration of these nodes deviates from the default configuration.
pub const MAX_SLOTS_TO_REMEMBER: Slot = 14400;
```

**Review by ebma — APPROVED** [review]
Looks good. We could already merge this even though the parameter is not accurate (ie. not matched by our deployed nodes). But the archive logic is broken anyways, so there is probably no harm done.

File: clients/vault/src/oracle/constants.rs (hunk 1 above)
Comment (b-yap) [reply clients/vault/src/oracle/constants.rs]: https://github.com/pendulum-chain/spacewalk/pull/355/commits/d7e87245e8192f6599e87e334ce650a8187f1e01
done

**Review by TorstenStueber — COMMENTED** [review]
👍

**Pull request merged**